project_description/README.rst

```diff
@@ -0,0 +1,66 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
```

Comment:
Update the link according to the [latest template](https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst)